Dockerfile.openhabian-aarch64-64bit

```diff
@@ -9,7 +9,7 @@ ENV INITSYSTEM on
 ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
 
 RUN apt-get update -qq \
-    && apt-get install -y -qq --no-install-recommends systemd systemd-sysv git wget python python-pip python3 python3-pip apt-utils jq virt-what \
+    && apt-get install -y -qq --no-install-recommends systemd systemd-sysv git wget python python-pip python3 python3-pip apt-utils jq virt-what gcc \
```

Comment:
1) Any hint why we need gcc here? Difference between raspbian and balena images?
2) Where do we have a feature with dependency on a compiler -> maybe better add the install there.
In case someone is running openhab on a Debian minimal install there might be no gcc installed.